builders/interoperability/xcm/remote-evm-calls.md

```diff
@@ -142,43 +142,29 @@ const decodedAddress = decodeAddress('INSERT_ADDRESS');
 }
 ```
 
-这是用于计算**multilocation衍生账户**的multilocation。您可以使用这个[计算**multilocation衍生账户**脚本](https://github.com/Moonsong-Labs/xcm-tools/blob/main/scripts/calculate-multilocation-derivative-account.ts){target=_blank}来帮助您获取它的值。您可以通过运行以下指令来获取：
+--8<-- 'text/xcm/calculate-multilocation-derivative-account.md'
 
-```sh
-yarn calculate-multilocation-derivative-account \
---w wss://wss.api.moonbase.moonbeam.network \
---a INSERT_MOONBASE_RELAY_ACCOUNT \
---p PARACHAIN_ID_IF_APPLIES \
---n westend
-```
-
-接着，让我们检查以上指令中输入的相关参数：
-
-- `-w`标志对应我们用于获得此信息的端点
-- `-a`标志对应您的Moonbase中继链账户地址
-- `-p`标志对应原链（如有）的平行链ID。如果您从中继链传送XCM则无需提供此参数
-- `-n`标志对应“westend”（Moonbase中继基于的中继链名称）的编码
-
-以我们的例子来说，我们将会通过Alice账户经由XCM传送远程EVM调用，也就是`5EnnmEp2R92wZ7T8J2fKMxpc1nPW5uP8r5K3YUQGiFrw8uG6`，因此指令和获得的结果将会如同下方图示。
+以Alice的中继链账户`5DV1dYwnQ27gKCKwhikaw1rz1bYdvZZUuFkuduB4hEK3FgDT`为例，您可以通过运行以下命令来计算他的Moonbase Alpha **multilocation衍生账户**：
 
 ```sh
 yarn calculate-multilocation-derivative-account \
---w wss://wss.api.moonbase.moonbeam.network \
---a 5EnnmEp2R92wZ7T8J2fKMxpc1nPW5uP8r5K3YUQGiFrw8uG6 \
---n westend
+--ws-provider wss://wss.api.moonbase.moonbeam.network \
+--address 5DV1dYwnQ27gKCKwhikaw1rz1bYdvZZUuFkuduB4hEK3FgDT \
+--parents 1
 ```
 
 所有数值被整理成以下表格：
 
 |            名称             |                                                                           数值                                                                            |
 |:---------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------:|
-|        原链编码地址         |                                                    `5EnnmEp2R92wZ7T8J2fKMxpc1nPW5uP8r5K3YUQGiFrw8uG6`                                                     |
-|        原链解码地址         |                                           `0x78914a4d7a946a0e4ed641f336b498736336e05096e342c799cc33c0f868d62f`                                            |
+|        原链编码地址         |                                                    `5DV1dYwnQ27gKCKwhikaw1rz1bYdvZZUuFkuduB4hEK3FgDT`                                                     |
+|        原链解码地址         |                                           `0x3ec5f48ad0567c752275d87787954fef72f557b8bfa5eefc88665fa0beb89a56`                                            |
 |        源链账户名称         |                                                                         `Westend`                                                                         |
```

Comment:
The Westend row can be removed (at least, that's how we handled it on the English site)

Reply:
good catch! thanks!